mesonbuild/interpreter.py

```diff
@@ -1161,6 +1162,11 @@ def run_method(self, args, kwargs):
     def get_id_method(self, args, kwargs):
         return self.compiler.get_id()
 
+    @noPosargs
+    @permittedKwargs({})
+    def get_linker_id_method(self, args, kwargs):
```

Comment:
This requires a `@FeatureNewKwargs` check.

Reply:
Sorry, I meant `@FeatureNew`:

```suggestion
    @FeatureNew('compiler.get_linker_id', '0.53.0')
    def get_linker_id_method(self, args, kwargs):
```